tests/e2e/README.md

```diff
@@ -0,0 +1,25 @@
+# End-to-End Testing
+
+## Context
+
+The FSx OpenZFS CSI driver utilizes a variety of workflows to validate end to end functionality in a Kubernetes environment.
+The framework and existing testsuites for Kubernetes storage E2E testing is leveraged to test common cases.
+Untested or unique test cases are also tested using a custom testsuite.
+
+Due to the nature end-to-end tests it is possible to leave testing artifacts stranded.
+Events such as abrupt termination or server-side API failures can prevent resources from being deleted properly.
+The `fsx_delete_e2e_resources` hack cleans up all testing resources that may have been left stranded.
+
+## Running
+
+In order to run the E2E tests, an EKS cluster with an OpenZFS driver installed must be utilized.
+Follow the respective section depending on if a test environment is already set up.
```

Comment:
There's only one section here right now, do we plan on adding a method to run the tests if the environment is not already set up properly?